extensions/smallrye-reactive-messaging/runtime/src/main/java/io/quarkus/smallrye/reactivemessaging/runtime/QuarkusWorkerPoolRegistry.java

```diff
@@ -1,10 +1,11 @@
 package io.quarkus.smallrye.reactivemessaging.runtime;
 
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Optional;
+import java.lang.reflect.InvocationTargetException;
+import java.util.*;
```

Comment:
No star imports.

Reply:
Rebased and push forced into this branch after #34716